src/RcppExports.cpp

```diff
@@ -18,6 +18,28 @@ BEGIN_RCPP
     return rcpp_result_gen;
 END_RCPP
 }
+// svmlin_rcpp
+List svmlin_rcpp(S4 X, NumericVector y, int l, int algorithm, double lambda, double lambda_u, int max_switch, double pos_frac, double Cp, double Cn, NumericVector costs, bool verbose);
```

Comment:
Other C++ functions are prefixed differently. Better to make all of them `qatd_cpp_*` or `*_rcpp`.

Reply:
Hey I got those files as a birthday gift! 😃

Reply:
@koheiw I've implemented the other changes, would you like to make this change, or leave it for a (separate) global change across **quanteda** and **quanteda.textmodels**?

Reply:
Yes, I will make changes at once.